deployments/intersmash-deployments-shared/intersmash-deployments-shared-wildfly/wildfly-bootable-jar-javax/pom.xml

```diff
@@ -93,4 +93,38 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
```

Comment:
Here maybe we'd want to do something different. This Bootable JAR deployment is based on legacy WildFly versions that are bound to pre Jakarta EE 9 related development, e.g.: 26.1.3, and eventually to the EAP 7.z stream. I'd suggest to remove this last change, I am planning to adjust that deployment very soon anyhow.

Reply:
done: I just left an "empty" configuration as a reminder to propagate channels info in some future